_posts/2016-07-27-highlights-of-yast-development-sprint-22.md

```diff
@@ -0,0 +1,155 @@
+---
+layout: post
+date: 2016-07-27
+title: Highlights of YaST development sprint 22
+category: SCRUM
+tags:
+- Distribution
+- Factory
+- Programming
+- Systems Management
+- YaST
+---
+
+[openSUSE Conference’16][https://events.opensuse.org/conference/oSC16],
```

Comment:
Already fixed in another branch, I'll keep it as it is now to avoid conflicts and open a new PR with the fix.